string plugin
string sensor_id
float64 timestamp
string description
map<string,string> metadata